src/Unleash/Strategies/StrategyUtils.cs

```diff
@@ -12,14 +12,14 @@ internal class StrategyUtils
         /// <summary>
         /// Takes to string inputs concat them, produce a hashCode and return a normalized value between 0 and 100;
         /// </summary>
-        public static int GetNormalizedNumber(string identifier, string groupId, int normalizer = 100)
+        public static int GetNormalizedNumber(string identifier, string groupId, uint randomSeed, int normalizer = 100)
```

Comment:
```suggestion
        public static int GetNormalizedNumber(string identifier, string groupId, uint randomSeed = 0, int normalizer = 100)
```
Can you just default to zero?

Reply:
I went with what Chris implemented in the Java client. Open for discussion, but forcing a choice was intentional:
https://github.com/Unleash/unleash-client-java/pull/220

Reply:
Good, I considered that possibility